src/mango_idx_text.erl

```diff
@@ -226,6 +227,9 @@ construct_analyzer({Props}) ->
     end.
 
 
+%% Empty Selector
+indexable_fields({[]}) ->
+    [];
```

Comment:
You'll need to be more specific. As I recall, an empty selector is a specific error we test for and reject, and if not, it sounds like somethign we should test and reject early.